<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreEditor</class>
 <widget class="QWidget" name="ScoreEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>427</width>
     <height>126</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Note">
    <attribute name="title">
     <string>Note</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonDo">
       <property name="text">
        <string>Do</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonRe">
       <property name="text">
        <string>Re</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="buttonFa">
       <property name="text">
        <string>Fa</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonMi">
       <property name="text">
        <string>Mi</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonSol">
       <property name="text">
        <string>Sol</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="buttonLa">
       <property name="text">
        <string>La</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonSi">
       <property name="text">
        <string>Si</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="buttonPause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Accidental">
    <attribute name="title">
     <string>Accidental</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonSharp">
       <property name="text">
        <string>Sharp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="buttonDoubleSharp">
       <property name="text">
        <string>Double Sharp</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonFlat">
       <property name="text">
        <string>Flat</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonDoubleFlat">
       <property name="text">
        <string>Double Flat</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="buttonAucun">
       <property name="text">
        <string>Aucun</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Octave">
    <attribute name="title">
     <string>Octave</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonNegatif">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Duration">
    <attribute name="title">
     <string>Duration</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonWhole">
       <property name="text">
        <string>Whole</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonHalf">
       <property name="text">
        <string>Half</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonQuarter">
       <property name="text">
        <string>Quarter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button16th">
       <property name="text">
        <string>16th</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="button8th">
       <property name="text">
        <string>8th</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="buttonLonga">
       <property name="text">
        <string>Longa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonDot">
       <property name="text">
        <string>Dot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button32th">
       <property name="text">
        <string>32th</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="buttonDoubleDot">
       <property name="text">
        <string>Double dot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="buttonHalfTriplet">
       <property name="text">
        <string>Half Triplet</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="buttonQuarterTriplet">
       <property name="text">
        <string>Quarter Triplet</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button8thTriplet">
       <property name="text">
        <string>8th Triplet</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="buttonQuintuplet">
       <property name="text">
        <string>Quintuplet</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="previewnote" native="true">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>9</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAjouterNote">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter note</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
